### ISSUE TITLE
Kubernetes support

### ISSUE THREAD

**Issue by arcastro** [tegonhq/docker#1]
Hello, 

is the [coordinator](https://github.com/tegonhq/docker/blob/main/docker-compose.yaml#L178-L195) service required for self-hosting?

Dependency on the docker api prevents those looking to self-host on Kubernetes

> volumes:
>      - /var/run/docker.sock:/var/run/docker.sock

Looking forward to kubernetes support down the road if possible

**Comment by harshithmullapudi** [comment]
Yeah you are right, we are working with trigger.dev to get a Kubernetes support, it should be supported we will add to the roadmap
